src/pages/react-redux-training-amsterdam.js

```diff
@@ -0,0 +1,177 @@
+import header from '../components/layout/Header.json'
+import { InstallmentsCard, PaymentSection } from '../components/payment'
+import { Link, Breadcrumb } from '../components/navigation'
+import { selectFirstTraining, PART_TIME, AMSTERDAM } from '../config/data'
+import { LIST_TWO_COL } from '../components/curriculum/selectCurriculumLayout'
+
+const training = selectFirstTraining(PART_TIME, AMSTERDAM)
+
+const PartTimeAmsterdam = () => (
+  <React.Fragment>
+    <Breadcrumb
+      path={[
+        { to: '/', label: 'Home' },
+        { to: '/react-redux-graphql-part-time-course', label: 'Part-time' },
+        { to: '/react-redux-training-amsterdam', label: 'Amsterdam' },
+      ]}
+    />
+    <Header
+      titleLines={[
+        'React Redux part-time course',
+        `${training.dates} - Amsterdam`,
+      ]}
+      subtitle="Take your dev career to the next level by mastering<br />React and Redux - without missing a day at work!"
+      links={header.landingTraining.links}
+      bgImg="part-time"
+    />
+    <TopSection xsBgDark>
+      <Grid>
+        <Card bg="dark">
+          <Row>
+            <Col xs={12} md={6} lg={5} lgOffset={1}>
+              <PaymentSection
+                data={{
+                  trainingInstanceId: training.trainingInstanceId,
+                  price: training.price,
+                  discountPrice: training.discountPrice,
+                  priceGoesUpOn: training.priceGoesUpOn,
+                  currency: training.currency,
+                }}
+              />
+              <InstallmentsCard price={1500} />
+            </Col>
+            <Col xs={12} md={6} lg={4} lgOffset={1}>
+              <Video youtubeID="E_4eQQHjc7A" />
+              <TrainingDetails
+                date={training.dates}
+                timing="9am - 6pm Friday's"
+                location={
+                  <React.Fragment>
+                    {training.location}
+                    {/* <Link to="https://www.google.com/maps/place/Makers/@51.5173403,-0.0754695,17z/data=!3m1!4b1!4m5!3m4!1s0x48761caf26599a83:0x9b451d586c649129!8m2!3d51.5173403!4d-0.0732808">
+                      See on map
+                    </Link> */}
+                  </React.Fragment>
+                }
+                coaches={[ROY_DERKS, ALEX_LOBERA]}
+              />
+              <Row>
+                <Newsletter />{' '}
+              </Row>
+            </Col>
+          </Row>
+        </Card>
+      </Grid>
+    </TopSection>
+    <Section>
+      <Grid>
+        <Row>
+          <HideComponentsUsingCss xs sm>
+            <Col md={6} lg={5}>
+              <Image
+                src={PART_TIME_IMG}
+                width="100%"
+                alt="ReactJS Academy coach Alex assists a student, being next to them, inspecting their code and helping them on their learning path."
+              />
+            </Col>
+          </HideComponentsUsingCss>
+          <Col md={6} lg={5} lgOffset={1}>
```

Comment:
@ric9176 should we create a component for this PartTimeTargetAudience? and use it London, Barcelona, and Lisbon? otherwise, we have the same text in 3 places